DCPvsOID/DCPvsOID.md

```diff
@@ -51,20 +51,24 @@ As participants (e.g. through M&A activities) and policies evolve or new partici
 ### Use case conclusion
 In the context of Manufacturing-X, decentralized identities and verifiable credentials empower organizations like Company A and Company B to exchange data securely and transparently. By leveraging protocols such as the Eclipse DCP, these entities not only adhere to the requirements of the Digital Product Passport but also foster a collaborative environment rooted in trust and compliance. As Europe moves toward the mandatory implementation of DPPs under the ESPR, such technologies will play a crucial role in realizing the vision of Manufacturing-X.
 
-## Introduction to Decentralized Claims in Dataspaces
-The issuance, presentation and verification of decentralized claims requires a protocol for securely sharing decentralized identities and credentials. It helps ensure that data access is secure by verifying identities and credentials without relying on a central authority.
+## Introduction to Verifiable Credentials in Dataspaces
+The issuance, presentation and verification of verifiable credentials requires a protocol for securely sharing policies, claims and evidence.
```

Comment:
Please justify evidence. This is about the protocol and not about a Trust Framework. Otherwise remove.

Reply:
The protocol are for exchanging verifiable credentials (not claims)

The claims are inside the exchanged verifiable credentials. So are [evidence](https://www.w3.org/TR/vc-data-model-2.0/#evidence).

What Trust Framework ?